src/main/resources/hudson/security/DangerousMatrixPermissionsAdministrativeMonitor/message.groovy

```diff
@@ -25,21 +25,23 @@
 package hudson.security.DangerousMatrixPermissionsAdministrativeMonitor
 
 def f = namespace(lib.FormTagLib)
-
-div(class: "error") {
+def foo = it
```

Comment:
Can you use an explicit variable name ? I don't know what foo is just by reading it. Why is it needed, isn't `it` working ?

Reply:
Fixed, i had to use the variable to deal with a groovy scope issue